ui/README.md

```diff
@@ -25,15 +25,15 @@ already provides it.
 
 Your demo should run on whichever web browser you prefer.
 
-## Craft Demo Presentation Outline
+## Candidate Exercise Presentation Outline
 
 We have created a PowerPoint template which covers an outline of what we would
 like to talk about during the first section of your interview before you share
 your craft demo. Feel free to use the template provided or make your own. For
 developers, we are more interested in your code than your PowerPoint skills, so
 plan your prep time accordingly.
 
-[PowerPoint Template and Craft Demo Outline](https://github.com/QuickBase/interview-demos/blob/master/ui/QuickBase_CraftDemo_PresentationTemplate.pptx)
+[PowerPoint Template and Craft Demo Outline](https://github.com/QuickBase/interview-demos/blob/gbowne-quickbase-patch-1/ui/Quickbase_CandidateExercise_PresentationTemplate.pptx)
```

Comment:
This link shouldnt need to change. You are just updating the ppt at the same location right?

Reply:
I changed the name of the file to reflect Quick Base -> Quickbase and Craft Demo -> Candidate Exercise

Reply:
Yep - you'll probably need to point to **master**/ui/Quickbase_CandidateExercise_PresentationTemplate.pptx instead of **gbowne-quickbase-patch-1**/ui/Quickbase_CandidateExercise_PresentationTemplate.pptx.